aiogram_dialog/widgets/kbd/force_reply.py

```diff
@@ -0,0 +1,30 @@
+from typing import Callable, Optional, Dict, Awaitable
+
+from aiogram.types import CallbackQuery, ForceReply as ForceReplyMarkup
+
+from aiogram_dialog.dialog import Dialog
+from aiogram_dialog.manager.manager import DialogManager
+from aiogram_dialog.widgets.text import Text
+from .base import Keyboard
+from ..when import WhenCondition
+
+OnClick = Callable[[CallbackQuery, "Button", DialogManager], Awaitable]
+
+
+class ForceReply(Keyboard):
```

Comment:
If it is KEyboard, It must be compatible with other keyboards. E.g. can be combined in Group and MUST follow Liskov Substitution Principle